sycl/plugins/esimd_cpu/pi_esimd_cpu.cpp

```diff
@@ -268,9 +417,10 @@ template <class KernelType, class KernelArgType, int DIMS> class libCMBatch {
 
     SpaceDim[0] = (uint32_t)Range[0];
 
-    PLACEHOLDER_UNUSED(WrappedLambda_ID_1DIM);
-    // TODO : Invoke invokeLambda_ID_1DIM through CM's multi-threaded
-    // kernel launching with WrappedLambda_ID_1DIM and dimension info
+    ESimdCPUKernel ESimdCPU((fptrVoid)InvokeLambda_ID_1DIM, GroupDim, SpaceDim);
+
```

Comment:
also, instead of duplicating code, template runInterationSpace functions by NDims

Reply:
is hwThreads field used?

Reply:
I'll apply templating `runIterationSpace` with `InvokeLambda` functions after the ESIMD_CPU support becomes fully functional in the sycl branch.

Reply:
`hwThreads` removed. `runIterationSpace` and `InvokeLambda` are updated with templated functions.